### DISCUSSION TITLE
The form Steps not getting updated in the form Designer on click

### DISCUSSION THREAD

**Discussion by Harshtone** [Budibase/budibase#5921] [General]
Hi,
I was working with the Form Steps, I only see the first step of the form in the Designer being updated. if i click on the other steps the designer still stays at the first step . so its been little difficult for me to design other steps without preview it. in the designer . please let me know if others facing this problem as well. thanks 
![Screenshot (19)](https://user-images.githubusercontent.com/55141711/168749173-45137552-d1bd-4bcb-b2ff-453cee448a59.png)

**Comment by xnohat** [comment]
I have faced this problem, you can change Step value box to 1 , all step will show beside others, you can easy design it, when complete, change Step value to correct order

**Comment by melohagan** [comment]
Thank you @xnohat - that's a very nice workaround :)

I've also raised a minor bug: https://github.com/Budibase/budibase/issues/5935
